TodoList/src/app/screens/List/index.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types';
+import {
+  View,
+  Text,
+  ScrollView,
+  StyleSheet,
+  TouchableOpacity
+} from 'react-native'
+
+import Checkbox from '../Checkbox'
+import { styles } from './styles.js'
+
+export default class List extends Component {
+  renderItem = (item, i) => {
+    const {onToggleItemCompleted, onRemoveItem} = this.props
+    const itemStyle = item.completed ? [styles.item, styles.completed] : styles.item
+    return (
+      <View key={i} style={itemStyle}>
+        <Text> {item.label} </Text>
+        <View style={styles.section}>
+          <Checkbox
+            isChecked={item.completed}
+            onToggle={() => onToggleItemCompleted(i)}
+          />
+          <TouchableOpacity onPress={() => onRemoveItem(i)}>
+            <Text style={styles.removed}> &times; </Text>
+          </TouchableOpacity>
+        </View>
+      </View>
+    )
+  }
+
+  render() {
+    const {items} = this.props
+    return (
+      <ScrollView style={styles.container}>
```

Comment:
You don't need `ScrollView` for this. Using `FlatList` is more appropriate, since you have an array of items to map.